tools/onnx_subgraph/src/lib/partition.cpp

```diff
@@ -21,6 +21,62 @@
 
 #define MAX_DEPTH 1000
 
+/**
+ * Prints the subgraph information of an ONNX model to specified files.
+ *
+ * @param Subgraphs A vector containing subgraph information.
+ * @param SubgraphFileName The filename for the output of subgraph information.
+ * @param OtherSubgraphs A vector containing other subgraph information.
+ * @param OtherSubgraphFileName The filename for the output of other subgraph information.
+ */
+void PrintSubgraphs(std::vector<onnx::GraphProto> Subgraphs, char *SubgraphFileName,
+                    std::vector<onnx::GraphProto> OtherSubgraphs, char *OtherSubgraphFileName)
```

Comment:
Q) can we use like

```suggestion
void PrintSubgraphs(std::vector<onnx::GraphProto> &Subgraphs, const char *SubgraphFileName,
                    std::vector<onnx::GraphProto> &OtherSubgraphs, const char *OtherSubgraphFileName)
```
passing vector will use more memory.

and

for variable name, instead of `Subgraphs` how about using `subgraphs`
and using `otherSubgraphs` instead of `OtherSubgraphs`
and also for names variable.